sw/device/tests/ast_clk_outs_test.c

```diff
@@ -56,20 +57,10 @@ bool test_main(void) {
   CHECK_DIF_OK(dif_aon_timer_init(
       mmio_region_from_addr(TOP_EARLGREY_AON_TIMER_AON_BASE_ADDR), &aon_timer));
 
-  // TODO(#13611) Uncomment when fixed
```

Comment:
The PR mentioned in that issue as a possible fix, https://github.com/lowRISC/opentitan/pull/13461, is now merged -- might be worth another try.

Reply:
I rebased and changed the BUILD file to declare these tests as not "broken" for cw310, and the tests timeout running on the fpga... So it is not quite fixed.